cmd/ncproxy/store.go

```diff
@@ -0,0 +1,84 @@
+package main
+
+import (
+	"context"
+
+	"github.com/containerd/containerd/errdefs"
```

Comment:
Can we make our own error definitions instead of using containerd's?

Reply:
is there a particular reason you would prefer this?

Reply:
I think we should just define our own errors here. Using containerd's doesn't make sense unless our error returns need to be interoperable with containerd errors.